Wikipedia/Code/WKWebView+LoadAssetsHtml.m

```diff
@@ -25,8 +25,31 @@ - (void)loadHTML:(NSString*)string withAssetsFile:(NSString*)fileName scrolledTo
     }
     
     WMFProxyServer *proxyServer = [WMFProxyServer sharedProxyServer];
+    
+    
+    NSUInteger windowPixelWidth = (NSUInteger)(MIN(self.bounds.size.width, self.bounds.size.height) * self.window.screen.scale);
+    
+    NSUInteger targetImageWidth = 320;
+    
+    if (windowPixelWidth < 640) {
+        targetImageWidth = 320;
+    } else if (windowPixelWidth < 800) {
+        targetImageWidth = 640;
+    } else if (windowPixelWidth < 1024) {
+        targetImageWidth = 800;
```

Comment:
Actually, now that I think of it, we should probably just use the same image scale logic we use to determine other image widths...

How about we add a method like the following to UIScreen+WMFImageWidth.m

    - (NSNumber*)wmf_articleImageWidthForScale {
        return @(self.wmf_maxScale * WMFImageWidthMedium);
    }

This would return 320 for 1x density screens, and 640 for 2x. Then in the future if we want to enable 3x we could just update wmf_maxScale and we'd get the change everywhere - in both articles *and* native code. This would also let us expose a user preference for low bandwidth by letting the user change wmf_maxScale, (should prob wait on adding that that for now).